### PR TITLE
fixed broken tests and add github action workflow

### PR DESCRIPTION
**Pull request by Yzm1234**

### CODE REVIEW

**Review by sychan — APPROVED** [review]
This one looks good, merging.

File: kbase.yml [hunk 1]
```diff
@@ -8,7 +8,7 @@ service-language:
     python
 
 module-version:
-    0.0.5
+    0.0.6
 
 owners:
     [jjeffryes]
```
Comment (sychan) [inline kbase.yml]: Please add yourself and dylan as addition owners. jjeffreyes left KBase quite some time ago.

**Review by sychan — CHANGES_REQUESTED** [review]
Please fix the owners in kbase.yml and then I'll merge it.

File: kbase.yml (hunk 1 above)
Comment (Yzm1234) [reply kbase.yml]: Added

**Pull request merged**